app/cdap/components/GridTable/components/AaToolbar/images.tsx

```diff
@@ -0,0 +1,242 @@
+/*
```

Comment:
Change the name of this file to iconsStore

Reply:
Change the name of this component to TransformationsToolbar

Reply:
Done